src/components/TxData.js

```diff
@@ -146,7 +147,8 @@ class TxData extends React.Component {
     });
 
     try {
-      const verificationData = await this.props.wallet.graphvizNeighborsQuery(
+      const wallet = getGlobalWallet();
+      const verificationData = await wallet.graphvizNeighborsQuery(
```

Comment:
```suggestion
      const { graphvizNeighborsQuery } = getGlobalWallet();
      const verificationData = await graphvizNeighborsQuery(
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.